docs/audit_report/src/side_channels/01_03_ml_kem.rst

```diff
@@ -0,0 +1,65 @@
+""""""
+ML KEM
```

Comment:
The specification writes the algorithm with a hyphe.
```suggestion
ML-KEM
```
Let's apply this to the other `ML KEM`s in this chapter as well.

Reply:
Updated, thanks :+1: